cloudsync/tasks.py

```diff
@@ -211,11 +212,17 @@ def update_video_statuses(self):
             refresh_status(video)
         except EncodeJob.DoesNotExist:
             # Log the exception but don't raise it so other videos can be checked.
-            log.exception("No EncodeJob object exists for video id %d", video.id)
+            log.exception("No EncodeJob object exists for video id %d",
+                          positional_args=(video.id,),
+                          video_id=video.id)
             video.update_status(error)
         except ClientError as exc:
             # Log the exception but don't raise it so other videos can be checked.
-            log.exception("AWS error when refreshing job status for video %d: %s", video.id, exc.response)
+            log.exception("AWS error when refreshing job status for "
+                          "video %d: %s",
+                          positional_args=(video.id, exc.response),
+                          video_id=video.id,
+                          response=exc.response)
```

Comment:
```python
log.exception("AWS error when refreshing job status", video_id=video.id)
```

Reply:
I'll take out the positional_args, but I'll keep the response as a structured log field so that we can have it in Kibana.

Reply:
Changed in 17e0a9e